template/core/domain/src/main/java/io/bloco/core/domain/models/Book.kt

```diff
@@ -0,0 +1,15 @@
+package io.bloco.core.domain.models
+
+import io.bloco.core.data.models.BookDto
+
+data class Book(
+    val key: String,
+    val title: String
+)
+
+fun BookDto.toModel(): Book = Book(
+    key = key.removePrefix(Key_Prefix),
+    title = title
+)
+
+private const val Key_Prefix = "/works/"
```

Comment:
Why do we need to remove the prefix?

Reply:
@LilianaFaustinoBloco

Reply:
This key is the ID that we use to request book details and it comes with this prefix from API.

I decided to remove it for consistency, so as not to be saving a useless piece of the string. Still, we might not remove and just change the book detail request path (that is `/works/$id.json`).